tests_e2e/test_error_handling.py

```diff
@@ -91,3 +92,40 @@ def test_corrupted_config_in_default_location(
     )
     assert result_healthy.returncode == 0, result_healthy.stderr
     assert "dev" in result_healthy.stdout and "integration" in result_healthy.stdout
+
+
+@pytest.mark.e2e
+def test_initial_log_with_loaded_external_plugins_in_custom_log_path(
+    snowcli, temp_dir, isolate_default_config_location, test_root_path
+):
+    default_config = Path(temp_dir) / "config.toml"
+    default_config.write_text(
+        dedent(
+            f"""
+    [cli.logs]
+    path = "{temp_dir}/logs"
```

Comment:
Maybe I'm wrong but I think this is the same path as the default one in this case (`temp_dir / logs`), so this test isn't testing what it says :)

Reply:
FYI this path style is platform dependent, we should use os.path.join

Reply:
I changed path and used os.path.join